common/djangoapps/util/tests/test_markup.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for util.markup
+"""
+
+import unittest
+
+import ddt
+
+from edxmako.template import Template
+from util.markup import escape, HTML, ugettext as _, ungettext
```

Comment:
@dianakhuang @cahrens You can unblock this.  I spoke with @nedbat.

Here's what was decided:
1. We will add all these methods so we can have a single import like above (i.e. a central place for all these helpers)
2. We will not escape in the ugettext and ungettext methods.  Instead, we will follow the same approach as everywhere else where we will either add a filter to the line, working toward escaping at the page level, and ultimately for the whole system.

My comments should stand around including the "| h" filter for now.  Unless we want to point to a doc where we can adjust this over time, or explain our full strategy for escaping in Mako.